File: Server/Sinatra/adminkun.rb

class Adminkun
  attr_accessor :serial_number
  attr_accessor :index_title
  attr_accessor :index_image_url
  attr_accessor :index_overview
  attr_accessor :body_url
  attr_accessor :body_image_url

  def initialize #(args)
    self.serial_number = 0.to_i
    self.index_title = ""
    self.index_image_url = ""
    self.index_overview = ""
    self.body_url = ""
    self.body_image_url = ""
  end
end


class MrAdmin
  attr_accessor :serial_number
  attr_accessor :index_title
  attr_accessor :index_image_url
  attr_accessor :index_overview
  attr_accessor :body_url
  attr_accessor :body_image_url

  def initialize #(args)
    self.serial_number = 0.to_i
    self.index_title = ""
    self.index_image_url = ""
    self.index_overview = ""
    self.body_url = ""
    self.body_image_url = ""
  end
end

File: Server/gaeo/application/model/adminkun.py
from google.appengine.ext import db
from gaeo.model import BaseModel, SearchableBaseModel

class Adminkun(BaseModel):
  serial_number = db.IntegerProperty(required = True)
  read_flag = db.BooleanProperty(required = True)
  index_title = db.StringProperty(required = True)
  index_overview = db.TextProperty(required = True)
  index_image_url = db.LinkProperty(required = True)
  body_url = db.LinkProperty(required = True)
  body_image_url = db.LinkProperty(required = False)
  lang = db.StringProperty(required = False)
  post_at = db.DateTimeProperty(required=True, auto_now_add=True)
  pass
  
